server/memphis_handlers_stations.go

```diff
@@ -108,6 +108,8 @@ func removeStationResources(s *Server, station models.Station) error {
 		return err
 	}
 
+	DeleteTagsByStation(station.Name)
```

Comment:
you can send the ID instead, then you won't need to get the station again inside this function